packages/adapter/src/utxo/bitcoin/bitcoin.ts

```diff
@@ -27,6 +28,22 @@ export const BTCGetAddress = async (
   }
 }
 
+export const BTCGetPublicKeys = async (
```

Comment:
This guy was missing and so was `btc_getPublicKeys` and its handler - this calls the `btc_getPublicKeys` with the same abstraction other methods in this module do so